### ISSUE TITLE
Lato font being used?

### ISSUE THREAD

**Issue by aried3r** [gettalong/hexapdf-website#1]
Hey! First of all, thanks for hexapdf and its extensive documentation!

tl;dr: Lato isn't being used, but if you happen to have Lato fonts installed locally, the website fonts are very thin.

While reading through the documentation I had the – subjective – impression that fonts were too thin.

I checked which fonts it used.

https://github.com/gettalong/hexapdf-website/blob/062779686f6d17f70f29c797f4e514d9b39a2a4d/src/assets/css/site.css#L1

While this repo contains the Lato font files, it doesn't seem like they are loaded by the website. I happen to have them installed on my system so Lato ended up being used. I checked with different `font-weight`s and here are some screenshots when I change the font-weight. On my system, 300 seems even thinner than 100.

| 100 | 300 | 400 |
|-----|-----|-----|
| <img width="778" alt="Screenshot 2022-03-01 at 15 20 30" src="https://user-images.githubusercontent.com/1301152/156186249-d26687f1-c0f2-40e8-94f4-ae841f8b1a6d.png"> | <img width="779" alt="Screenshot 2022-03-01 at 15 20 15" src="https://user-images.githubusercontent.com/1301152/156186286-e62fe6fa-2f60-4644-905c-160cec114e61.png"> | <img width="779" alt="Screenshot 2022-03-01 at 15 20 43" src="https://user-images.githubusercontent.com/1301152/156186321-21a20802-d6c8-43e8-8199-f1af314255da.png"> |

Then I uninstalled Lato from my system to check, and indeed, it now renders better:

<img width="836" alt="Screenshot 2022-03-01 at 15 33 16" src="https://user-images.githubusercontent.com/1301152/156188588-8702ea5f-706b-4d64-8fed-376531dc1ab4.png">

However as you can see, it doesn't render using Lato, but rather the [System Font Stack](https://css-tricks.com/snippets/css/system-font-stack/) that also GitHub uses.

So while this only renders wrongly on my system because I had Lato installed locally, perhaps the CSS should switch to only using system fonts or include Lato when loading the website. WDYT?

**Comment by gettalong** [comment]
Thank you!

It's a bit strange than the font looks thicker on your system with font weight 100. I just tried it and here at least the font weight 100 is clearly much thinner than font weight 300.

As for the missing inclusion of the Lato fonts: This was three years ago but I think this was just an oversight, possibly also because I have Lato installed as system font. I will test a bit and see what would be better.

**Comment by gettalong** [comment]
@aried3r I have included the missing Lato fonts and now I'm using the correct one for weight 300: before it was Lato-Thin whereas Lato-Light is actually the correct one for weight 300. Can you have a look?

**Comment by aried3r** [comment]
Yep, looking good now!

![Screenshot 2022-03-07 at 13 28 34](https://user-images.githubusercontent.com/1301152/157034678-d5cd567b-6aa8-43a9-a3e2-a0a0ac825a4e.png)
